### PR TITLE
Add funding options

### PR DESCRIPTION
**Pull request by johnwatson484**
https://eaflood.atlassian.net/browse/SFI-2334﻿

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,52 @@ scripts/test
...
 ```
 
+## Funding codes
+
+In order to persist payment request data and construct a statement dataset, the service must be populated with a mapping of funding option codes to a name.
+These values are stored in the `fundingOptions` PostgreSQL database table.
+
+### SFI Pilot
+
+| Name | Code |
```
Comment (marctemp) [inline README.md:76]: would make the names case identical to the records in the db

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-statement-constructor",
-  "version": "0.6.0",
+  "version": "0.6.1",
```
Comment (marctemp) [inline package.json:3]: version is upto `0.7.0` as I speak 👀

Comment (marctemp) [reply package.json:3]: type rather

Comment (johnwatson484) [reply package.json:3]: Thanks will update.  I'm expecting merge conflicts with other pending tickets ahead of this one too so may need to make other changes.

File: README.md (hunk 1 above)
Comment (johnwatson484) [reply README.md:76]: Serves me right for copy and pasting the Pilot ones...

Fixed now.

File: README.md [hunk 3]
```diff
@@ -66,6 +66,52 @@ scripts/test
...
+| Improved grassland soils | 80004 |
+| Improved grassland | 80005 |
+| On farm woodland | 80006 |
+| Low and no input grassland | 80007 |
+| Water body buffering | 80008 |
+| Pilot participation payment | 80009 |
+
+### SFI
+
+| Name | Code |
```
Comment (marctemp) [inline README.md:90]: same case-thang for these

**Review by marctemp — APPROVED** [review]
Good to Go

**Review by marctemp — APPROVED** [review]
Approved

**Pull request merged**